mmcv/cnn/utils/weight_init.py

```diff
@@ -93,6 +93,10 @@ def bias_init_with_prob(prior_prob):
     return bias_init
 
 
+def _get_bases_name(m):
+    return [b.__name__ for b in m.__class__.__bases__]
```

Comment:
Just wondering, if there is multi level inheritance in PyTorch, e.g. C->B->A? Since the `__bases__` will only get the first level parent.

Reply:
yes, only get the first level parent. ``__bases__.__bases__`` is the second level